app/models/email_settings.rb

```diff
@@ -67,6 +74,14 @@ def generate_email_on_conf_updates(conference, user, conf_update_template)
     parse_template(conf_update_template, values)
   end
 
+  def generate_email_on_comment_create(conference, event, comment, user)
+    values = get_values(conference, user, event, comment)
+    comment_template = File.read("#{Rails.root}/app/views/admin/emails/comment_template.text.erb")
```

Comment:
Don't really like this approach of loading the view ...
@hennevogel what do you think? Any better idea?